### PR TITLE
Add portal back to lobby

### PR DESCRIPTION
**Pull request by frangauthier**

### CODE REVIEW

**Review by TheWhitcher — COMMENTED** [review]
yes

**Review by TheWhitcher — COMMENTED** [review]
looks good

File: Common/Scenes/Objects/magic_portal.gd [hunk 1]
```diff
@@ -18,10 +18,11 @@ func _process(delta):
 
 
 func _on_complete_button_body_entered(body):
-	var gem = gem_scene.instantiate() as Node3D
-	gem.scale = Vector3.ONE*0.003;
-	$GemLocation.add_child(gem)
-	Globals.complete_level(gem_color);
+	var new_gem = Globals.complete_level(gem_color);
```
Comment (frangauthier) [inline Common/Scenes/Objects/magic_portal.gd:21]: Well done.

**Comment by MunirKhaliqyar** [comment]
Nice commit

**Review by 2088826 — APPROVED** [review]
Approved.

**Review by MunirKhaliqyar — APPROVED** [review]

**Review by Baccast — APPROVED** [review]
very good

**Review by TheWhitcher — APPROVED** [review]

**Pull request merged**